README.md

```diff
@@ -1 +1,44 @@
-# mv-awesome-books
\ No newline at end of file
+# HTML-CAPSTONE-PROJECT
+
+> This project is a Microverse exercise on Javascript DOM manipulation, event listener, arrow function, let and const, Javascript Object, Arrays, String interpolation, local storage.
+
+
+![screenshot]()
+
+## Built With
+
+- Semantic HTML
+- JavaScript
+- Github workflow
+- Linters(Lighthouse, stylelint, Webhint, eslint)
+
+## Live Preview
+
+[Github deploy Page]()
```

Comment:
- [ ] Please keep in mind that you should either provide a valid live demo link or remove its section from the README since it's not a hard requirement for this project.
_Kindly notice that such a small mistake might leave a bad impression on future viewers._